docs/version-2.0/start-guide/installation/system-setup-xavier.md

```diff
@@ -16,19 +16,19 @@ ssh root@IP-ADDRESS
 ssh nv@192.168.10.46
 ```
 
-## Copy Autoware.Auto image to Xavier
+## Copy Autoware.Universe image to Xavier
 
 **NOTE**: docker should be installed with post-installation steps. For instructions please refer to:
 
 - [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu).
 - [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall).
 
-The docker image of Autoware.Auto is registered in [GitLab Container Registry](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/container_registry/2511358).
+The docker image of Autoware.Universe for arm64 is registered in [Autoware Foundation Container Registry](https://github.com/autowarefoundation/autoware/pkgs/container/autoware-universe/30821188?tag=galactic-20220728-prebuilt-cuda).
 
 1. Copy docker image to Xavier.
 
    ```console
-   docker pull registry.gitlab.com/autowarefoundation/autoware.auto/autowareauto/arm64/openadkit-foxy:latest
+   docker pull ghcr.io/autowarefoundation/autoware-universe:galactic-20220728-prebuilt-cuda
```

Comment:
Is there any reason to use such an old image?

```suggestion
   docker pull ghcr.io/autowarefoundation/autoware-universe:galactic-latest-prebuilt-cuda
```

Reply:
To be honest, I don't know the difference between the two images, when test on xavier, I pull the galactic-20220728-prebuilt-cuda image.

Reply:
`latest` is an alias for date-based images.
Generally, using the latest image is recommended. If you have to use old images for some reason, specifying the date is okay.

Reply:
For all tags, please refer to https://github.com/autowarefoundation/autoware/pkgs/container/autoware-universe/versions.

Reply:
The `ghcr.io/autowarefoundation/autoware-universe:galactic-20220728-prebuilt-cuda` image is also used to be consistent with [Minimum requirements](https://github.com/autowarefoundation/open-ad-kit-docs/blob/main/docs/version-2.0/start-guide/installation/index.md#minimum-requirements) in the tutorial.

This image is used as an example in many parts of the tutorial. for example `kubernetes yaml files`.

Reply:
Then it's okay to use `20220728` for this PR.
But since we've moved to Humble, I think we should consider updating the version at some timing.